mobile/src/actions/mediaBackup.js

```diff
@@ -45,7 +45,7 @@ export const mediaBackup = (dir) => async (dispatch, getState) => {
     const dirID = await getDirID(dir)
     const totalUpload = photosToUpload.length
     let uploadCounter = 0
-    for (let photo of photosToUpload) {
+    for (const photo of photosToUpload) {
```

Comment:
If you don't reassign the variable inside the loop, you can use `const` for the iterable variable. Cf. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...of#Examples